### PR TITLE
getPath failed on empty paths

### PR DESCRIPTION
**Pull request by mgttlinger**
Changed `head` to `headOption` to prevent crash on empty paths

### CODE REVIEW

File: core/src/main/scala/package.scala [hunk 1]
```diff
@@ -226,13 +226,13 @@ package object quiver {
 
   /** Find the first path in a search tree that starts with the given node */
   def getPath[N](v: N, t: RTree[N]): Option[Path[N]] =
-    t.find(_.head == v).map(_.reverse)
+    t.find(_.headOption == Some(v)).map(_.reverse)
```
Comment (timperrett) [inline core/src/main/scala/package.scala]: Small pet hate: can we possibly make `Some` -> `Option`. There are a few cases around `null` handling for which `Some.apply` is not safe.

**Comment by timperrett** [comment]
@mgttlinger thanks for the PR; one small comment :-) 

/cc @kaiserpelagic

**Comment by mgttlinger** [comment]
Sure thing, gonna change that later today.

**Comment by kaiserpelagic** [comment]
@mgttlinger @timperrett lgtm

**Comment by mgttlinger** [comment]
What is your release schedule? Aka when can I expect a release with that PR?

**Comment by timperrett** [comment]
@mgttlinger releases are automated on merge: https://repo1.maven.org/maven2/io/verizon/quiver/core_2.11/5.5.15-scalaz-7.1/
https://repo1.maven.org/maven2/io/verizon/quiver/core_2.11/5.5.15-scalaz-7.2/
etc

**Comment by mgttlinger** [comment]
That's great!

**Pull request merged**